prow/config/jobs.go

```diff
@@ -17,12 +17,62 @@ limitations under the License.
 package config
 
 import (
+	"fmt"
 	"regexp"
 	"time"
 
 	"k8s.io/api/core/v1"
 )
 
+// Preset is intended to match the k8s' PodPreset feature, and may be removed
+// if that feature goes beta.
+type Preset struct {
+	Labels       map[string]string  `json:"labels"`
+	Env          []kube.EnvVar      `json:"env"`
```

Comment:
does this compile? wrong import

Reply:
should updated to v1 now

Reply:
it was updated in the second commit. other than rebasing I didn't want to modify joe's commit.

Reply:
We are using go aliases now so kube.EnvVar is v1.EnvVar